src/main/java/marquez/api/models/DatasourceResponse.java

```diff
@@ -0,0 +1,21 @@
+package marquez.api.models;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class DatasourceResponse {
+
+  @JsonProperty("name")
+  String name;
+
+  @JsonProperty("createdAt")
+  String createdAt;
+
+  @JsonProperty("connectionUrl")
+  String owner;
```

Comment:
Should this be: `connectionUrl `?

Reply:
Yup - fixed in another PR which will be merged into this.

Reply:
Please see PRs #394, #387, and #384, where this is fixed.